apps/inviwo/packaging/packaging.cmake

```diff
@@ -52,10 +52,10 @@ if(IVW_PACKAGE_PROJECT)
     if(WIN32)
         # Need backslash for correct subdirectory paths with NSIS
         set(CPACK_PACKAGE_ICON              "${IVW_ROOT_DIR}\\\\resources\\\\inviwo\\\\inviwo_light.png")
-        set(CPACK_PACKAGE_INSTALL_DIRECTORY "Inviwo\\\\${IVW_VERSION}")
+        set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_PACKAGE_NAME}\\\\${IVW_VERSION}")

```

Comment:
What the reason for moving the out of the cmake folder?

Reply:
Because it is app-dependent. This cmake file is *only* used by the inviwo app. Minimals could have their own if we wanted to (or want to give an example).

Reply:
I think this matters most for MacOS bundles. It is not possible to start different executables in a bundle without explicitly going into the bundle